src/Asset/NpmManager.php

```
@@ -40,6 +81,109 @@ protected function getUpdateCommand(): string
+                continue;
+            }
+
+            $selectors[] = ProcessExecutor::escape('--workspace=' . $path);
+        }
+
+        if ([] === $selectors) {
+            throw $this->workspaceEnumerationFailure();
+        }
+
+        return $selectors;
+    }
```

PHPMD (2.15.0)

[warning] The method getWorkspaceSelectors() has a Cyclomatic Complexity of 14. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)